components/server/src/auth/verification-service.ts

```diff
@@ -54,35 +58,22 @@ export class VerificationService {
         return isPhoneVerificationEnabled;
     }
 
-    public markVerified(user: User): User {
-        user.lastVerificationTime = new Date().toISOString();
-        return user;
-    }
-
     public async verifyOrgMembers(organizationId: string): Promise<void> {
```

Comment:
is it fine that we are changing the logic here, are there DB tests for this service? If we are just moving functions without changing semantic is fine with me without adding tests, but if we start changing code we should rather start adding tests.

Reply:
It's just moving functions